packages/core/content-releases/admin/src/pages/tests/Releases.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import { ReleasesPage } from '../Releases';
+
+const render = () => ({
+  ...renderRTL(<ReleasesPage />, {
+    wrapper: ({ children }) => (
+      <ThemeProvider theme={lightTheme}>
+        <IntlProvider locale="en" messages={{}} defaultLocale="en">
+          {children}
+        </IntlProvider>
+      </ThemeProvider>
+    ),
+  }),
+  user: userEvent.setup(),
```

Comment:
I think you only need to setup the user once at the root of the file, then you can reuse the instance on all your tests